pkg/virt-controller/watch/vm/vm_test.go

```diff
@@ -6751,20 +6730,20 @@ var _ = Describe("VirtualMachine", func() {
 				Expect(err).To(Succeed())
 				Expect(vm.Status.Conditions).To(matcher, "restart Required")
 
-				if strat == &liveUpdate && len(fgs) > 0 {
+				if strat == &liveUpdate {
 					vmi, err = virtFakeClient.KubevirtV1().VirtualMachineInstances(vm.Namespace).Get(context.TODO(), vm.Name, metav1.GetOptions{})
 					Expect(err).ToNot(HaveOccurred())
 					Expect(vmi.Spec.Domain.CPU.Sockets).To(Equal(uint32(4)))
 				}
 			},
 				Entry("should appear if the feature gate is not set",
-					[]string{}, &liveUpdate, restartRequiredMatcher(k8sv1.ConditionTrue)),
+					&liveUpdate, restartRequiredMatcher(k8sv1.ConditionTrue)),
 				Entry("should appear if the VM rollout strategy is not set",
-					[]string{virtconfig.VMLiveUpdateFeaturesGate}, nil, restartRequiredMatcher(k8sv1.ConditionTrue)),
+					nil, restartRequiredMatcher(k8sv1.ConditionTrue)),
 				Entry("should appear if the VM rollout strategy is set to Stage",
-					[]string{virtconfig.VMLiveUpdateFeaturesGate}, &stage, restartRequiredMatcher(k8sv1.ConditionTrue)),
+					&stage, restartRequiredMatcher(k8sv1.ConditionTrue)),
 				Entry("should not appear if both the VM rollout strategy and feature gate are set",
```

Comment:
This `Entry` description should be renamed:
`"should not appear if the VM rollout strategy is set to LiveUpdate"`